server/server.js

```diff
@@ -117,4 +119,13 @@ app.get('/questions', (req, res) => {
     });
 });
 
+app.post('/interactions', (req, res) => {
+  atelierQueries.postInteraction(req.body, atelierHeaders)
+    .then(result => res.end(JSON.stringify(result)))
+    .catch(error => {
+      console.error(error);
```

Comment:
Same question about error throw/catch approach as before. Are we testing for success and missing flaws in our exception-handling because our async calls aren't failing often enough?

Reply:
Probably!